x-pack/plugins/cases/public/api/index.ts

```diff
@@ -43,3 +49,11 @@ export const getCasesStatus = async ({
 
   return convertToCamelCase<CasesStatusResponse, CasesStatus>(decodeCasesStatusResponse(response));
 };
+
+export const getCasesMetrics = async ({
```

Comment:
I can see in all of the other APIs we accept an abort signal, is there a particular reason to not require it here?

Reply:
I forgot it. Thanks!

Reply:
Done!